readthedocs/builds/managers.py

```diff
@@ -67,6 +72,12 @@ def create_latest(self, **kwargs):
         defaults.update(kwargs)
         return self.create(**defaults)
 
+    def get_object_or_log(self, **kwargs):
+        try:
+            return super(VersionManagerBase, self).get(**kwargs)
+        except ObjectDoesNotExist:
+            log.warning('Version not found for the pk = {pk}'.format(pk=kwargs.get('pk')))
```

Comment:
We should use `.format` when logging and should use `%s` instead.

Also, this method allows us to be called with different attributes than just `pk`. We should log all the ones used here.

Reply:
Thank you.
Learned something new.
I have pushed the changes.